inc/functions.php

```diff
@@ -57,7 +63,9 @@ function core_sitemaps_get_server() {
 }
 
 /**
- * Get a list of sitemaps.
+ * Gets a list of sitemaps.
```

Comment:
Maybe "Gets a list of sitemap providers." instead? That's a bit more precise and in line with the description for the other function below.

Reply:
👍

Reply:
Updated in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/173/commits/c2d3d4b4f1fed1ad34e041d9164c6bf071fb1d8e.